solr/test-framework/src/java/org/apache/solr/search/facet/DebugAgg.java

```diff
@@ -38,14 +38,12 @@ public ValueSource parse(FunctionQParser fp) throws SyntaxError {
       parses.incrementAndGet();
       final String what = fp.hasMoreArguments() ? fp.parseId() : "wrap";
 
-      switch (what) {
-        case "wrap":
-          return new DebugAgg(fp);
-        case "numShards":
-          return new DebugAggNumShards();
-        default: /* No-Op */
-      }
-      throw new RuntimeException("No idea what to do with " + what);
+      /* No-Op */
+      return switch (what) {
```

Comment:
I see you took a moment of pleasure to do Java 21 stuff :-)

Reply:
;-).  switch I like!